piControlMain.c

```diff
@@ -385,18 +391,12 @@ static int piControlReset(tpiControlInst * priv)
 		// ignore errors
 	}
 
-	if (piDev_g.machine_type == REVPI_CORE) {
-		PiBridgeMaster_Reset();
-	} else if (piDev_g.machine_type == REVPI_CORE_SE) {
-		PiBridgeMaster_Reset();
-	} else if (piDev_g.machine_type == REVPI_CONNECT) {
-		PiBridgeMaster_Reset();
-	} else if (piDev_g.machine_type == REVPI_CONNECT_SE) {
-		PiBridgeMaster_Reset();
-	} else if (piDev_g.machine_type == REVPI_COMPACT) {
+	if (piDev_g.machine_type == REVPI_COMPACT) {
 		revpi_compact_reset();
 	} else if (piDev_g.machine_type == REVPI_FLAT) {
 		revpi_flat_reset();
+	} else if (piDev_g.pibridge_rs485_supported == 1) {
+		PiBridgeMaster_Reset();
```

Comment:
This one sort of makes sense because the call to `PiBridgeMaster_Reset()` is really PiBridge-specific.